### PR TITLE
bug fix

### PR DESCRIPTION
**Pull request by javahuang**
1. fix(more formats toolbar): fix more formats toolbar can't sync with cell
2. fix mac delete button can't delete cell
3. fix can't delete cell with inline-style

### CODE REVIEW

**Review by mengshukeji — APPROVED** [review]
good job!

File: src/controllers/rowColumnOperation.js [hunk 1]
```diff
@@ -1629,6 +1629,9 @@ export function rowColumnOperationInitial(){
                         if(getObjType(d[r][c]) == "object"){
                             delete d[r][c]["m"];
                             delete d[r][c]["v"];
+                            if(d[r][c]["ct"] != null) {
+                                delete d[r][c]["ct"]["s"]
+                            }
```
Comment (wbfsa) [inline src/controllers/rowColumnOperation.js:1634]:
```suggestion
                            if(d[r][c]["ct"] != null) {
                                delete d[r][c]["ct"]["s"]
                            }
```
这里需要把d[r][c]["ct"].t == 'inlineStr' ,inlineStr重置为g，因为单元格为行内样式的话，会设置t为inlineStr，这里不重置会有隐患

**Review by wbfsa — CHANGES_REQUESTED** [review]

**Comment by mengshukeji** [comment]
感谢贡献，期待加入我们开发组，加这个微信：liurunze---

File: src/controllers/rowColumnOperation.js (hunk 1 above)
Comment (javahuang) [reply src/controllers/rowColumnOperation.js]:
```js
if(d[r][c]["ct"] != null && d[r][c]["ct"].t == 'inlineStr'){
  delete d[r][c]["ct"];
 }
```
刚看这个问题已经被别的同学修复了，我给这个删了

**Review by Dto161 — APPROVED** [review]

**Review by fylo5 — APPROVED** [review]

**Review by wbfsa — APPROVED** [review]

**Pull request merged**